grafana-plugin/src/containers/PluginConfigPage/PluginConfigPage.test.tsx

```diff
@@ -302,7 +302,7 @@ describe('PluginConfigPage', () => {
     // click the confirm button within the modal, which actually triggers the callback
     await userEvent.click(screen.getByText('Remove'));
 
-    await screen.findByTestId(successful ? PLUGIN_CONFIGURATION_FORM_DATA_ID : STATUS_MESSAGE_BLOCK_DATA_ID);
+    // await screen.findByTestId(successful ? PLUGIN_CONFIGURATION_FORM_DATA_ID : STATUS_MESSAGE_BLOCK_DATA_ID);
```

Comment:
Need some help on fixing this test properly now that the page is reloading on reset.

Reply:
hmm, this type of assertion _may_ no longer be possible for the `successful` scenario.

We're expecting the configuration form to be present here, but that is reliant on a full page/state reload to arrive at this state. It might be beyond what `@testing-library/react` and `@testing-library/user-event` are capable of? Curious to hear what someone from @grafana/grafana-oncall-frontend thinks here?